sunpy/coordinates/tests/test_transformations.py

```diff
@@ -0,0 +1,45 @@
+from sunpy.coordinates import Helioprojective
+
+import astropy.units as u
+from astropy.tests.helper import quantity_allclose
+
+import numpy as np
+
+
+def test_hpc_hpc():
+    rsun = 1*u.m
```

Comment:
You might as well add a comment acknowledging that `rsun` and `D0` are unphysical values here so that some future developer doesn't freak out.